### PR TITLE
Add CI to backend with Gitgub Actions

### PR DESCRIPTION
**Pull request by devdeM1**

### CODE REVIEW

File: .github/workflows/test_on_push.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    branches: '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout code
+    - uses: actions/checkout@master
```
Comment (Andmedoctopus) [inline .github/workflows/test_on_push.yml]: why master? Could you try to "fail" the CI

**Comment by Andmedoctopus** [comment]
Great
1. Add people to reviewers
1. Add .idea folder to `.gitignore`

**Review by Andmedoctopus — CHANGES_REQUESTED** [review]
remove .idea folder

File: .github/workflows/test_on_push.yml (hunk 1 above)
Comment (Andmedoctopus) [reply .github/workflows/test_on_push.yml]: Should works fine. It's probably version of actions

File: .github/workflows/tests_on_push.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    branches: '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout code
+    - uses: actions/checkout@CI
```
Comment (Andmedoctopus) [inline .github/workflows/tests_on_push.yml]: It doesn't work with `CI`

the branch name is the name of the checkout utility not of your branch. Just revert to master.

**Review by Andmedoctopus — APPROVED** [review]

**Review by vorobeybird — APPROVED** [review]

**Review by Zhental — APPROVED** [review]

**Pull request merged**